gradle.properties

```diff
@@ -0,0 +1,4 @@
+action.custom-1=checkMinecraftVersions
```

Comment:
This is a file for gradle to remember the project specific actions so that the IDE can show them to you in a menu. I can skip this for the commit if you want. I am not sure if this is globally valid or only for my IDE (Netbeans)

Reply:
I don't care, it looks fine.